client/asset/firo/firo.go

```diff
@@ -154,18 +164,29 @@ func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network)
 		NumericGetRawRPC:         false, // getrawtransaction uses either 0/1 Or true/false
 		LegacyValidateAddressRPC: true,  // use validateaddress to read 'ismine' bool
 		SingularWallet:           true,  // one wallet/node
-		UnlockSpends:             false, // checked after sendtoaddress
+		UnlockSpends:             true,  // does Not unlock coins after sendrawtransaction
 		AssetID:                  BipID,
 		FeeEstimator:             estimateFee,
 		ExternalFeeEstimator:     fetchExternalFee,
 		PrivKeyFunc: func(addr string) (*btcec.PrivateKey, error) {
-			return privKeyForAddress(w, addr)
+			return privKeyForAddress(exw, addr)
 		},
 	}
 
-	var err error
-	w, err = btc.BTCCloneWallet(cloneCFG)
-	return w, err
+	switch cfg.Type {
+	case walletTypeRPC, walletTypeLegacy:
+		var err error
+		exw, err = btc.BTCCloneWallet(cloneCFG)
+		return exw, err
+	case walletTypeElectrum:
+		// override Ports - no default ports
+		cloneCFG.Ports = dexbtc.NetPorts{}
+		// electrum Does have a getbalance RPC
+		cloneCFG.LegacyBalance = false
```

Comment:
`LegacyBalance` still has nothing to do with Electrum though.  Unsetting it has no purpose.  If you find it awkward to have it true for all wallet types, then you can set it to true in just `case walletTypeRPC, walletTypeLegacy`.